apps/admin/src/app/dashboard/components/Datatable.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import React from "react";
+
+const data = [
+  {
+    id: 1203,
+    name: "Erlich Bacham",
+    address: "New Baneshwor, Kathmandu",
+    date: "09/02/2024",
+    status: "Unverified",
+  },
+  {
+    id: 1203,
+    name: "Erlich Bacham",
+    address: "New Baneshwor, Kathmandu",
+    date: "09/02/2024",
+    status: "Unverified",
+  },
+  {
+    id: 1203,
+    name: "Erlich Bacham",
+    address: "New Baneshwor, Kathmandu",
+    date: "09/02/2024",
+    status: "Unverified",
```

Comment:
Status in backend is 
```
ACCEPTED, REJECTED and APPLIED
```
tehi anusar in frontend hai

Reply:
Got it